xikolo-ios/ViewControllers/Courses/VideoViewController.swift

```diff
@@ -155,7 +155,7 @@ extension VideoViewController: BMPlayerDelegate {
         if state == .bufferFinished {
             player.avPlayer?.rate = self.playerControlView.playRate  // has to be set after playback started
         } else if state == .playedToTheEnd {
-            TrackingHelper.sendEvent("VIDEO_END", resource: self.video, context: self.newTrackingContext)
```

Comment:
No, it is this here. If it is wrong, we can change it now.

Reply:
We have to check with @tobiasrohloff

Reply:
I don't think it is currently used in android: https://github.com/openHPI/xikolo-android/blob/b17d94966389fd4052de02e69fae8c132f6bb69b/app/src/main/java/de/xikolo/utils/LanalyticsUtil.java